.github/workflows/label-pr.yml

```diff
@@ -0,0 +1,12 @@
+name: "Label PRs"
+on:
+  - pull_request_target
```

Comment:
This might require changing permissions from 'read' to 'write' on the pull-requests. https://github.com/actions/labeler?tab=readme-ov-file#permissions
But maybe we already have that.